test/BUILD.bazel

```diff
@@ -38,25 +38,6 @@ cc_library(
     ],
 )
 
-cc_library(
```

Comment:
Please update the PR description: this PR no longer changes `backward`'s visibility